src/dbos-runtime/config.ts

```diff
@@ -247,6 +304,7 @@ export interface ParseOptions {
   appVersion?: string | boolean;
   silent?: boolean;
   forceConsole?: boolean;
+  userDbPoolSize?: number;
```

Comment:
A bit of a hack of `ParseOptions` to pass more user-provided info to `constructPoolConfig`